package.json

```
@@ -3,8 +3,8 @@
   "private": true,
   "version": "0.0.1",
   "scripts": {
-    "audit": "npm audit && lerna exec --concurrency 1 'echo $LERNA_PACKAGE_NAME && npm audit'",
-    "audit:fix": "npm audit fix && lerna exec --concurrency 1 'echo $LERNA_PACKAGE_NAME && npm audit fix'",
+    "audit": "npm audit & lerna exec --concurrency 1 'echo $LERNA_PACKAGE_NAME & npm audit'",
```

```suggestion
    "audit": "npm audit & lerna exec 'echo $LERNA_PACKAGE_NAME & npm audit'",
```
There's a risk that the messages get intermingled (with or without `--concurrency 1`), but since getting any error messages is enough of a sign that we should dig further I assum that's fine.  Anyway, since we're running parallel processes `lerna` might as well use concurrency.